docs/overview.rst

```diff
@@ -0,0 +1,165 @@
+- **Python venv**,
+- **Kubernetes**,
+- **Terraform**,
+- **Docker**,
+- **Container**,
+- **Node env**,
+- **Ruby venv**,
+- **Software Collections**.
+
+Those show up only if you enter a configured directory, or have configured environment variables.
+
+
+Known Limitations and Bugs
+--------------------------
+
+- Does not display the number of commits to be pushed in Mercurial repositories.
+- Browsing very large Subversion repositories may dramatically slow down
+  the display of Liquid Prompt (use :attr:`LP_DISABLED_VCS_PATHS` to avoid that).
+- Subversion repositories cannot display commits to be pushed because
+  that's not how Subversion works
+- The window's title escape sequence may not work properly on some terminals
+  (like `xterm-256`).
+- The analog clock requires a Unicode-aware terminal and at least a
+  sufficiently complete font on your system. The `Symbola <http://users.teilar.gr/~g1951d/>`_
+  font, designed by Georges Douros, is known to work well. On Debian or Ubuntu
+  install try the `fonts-symbola` or `ttf-ancient-fonts` package.
+  "Powerline" and "Nerd" fonts also have the approriate symbols.
+- The "sudo" feature is disabled by default as there is no way to detect
+  if the user has sudo rights without triggering a security alert
+  that will annoy the sysadmin.
+
+
+Competitors
+-----------
+
+All prompt systems tend to focus on some feature sets.
+If you don't like Liquid Prompt's design, you may be interested in one of those popular prompts:
+
+- `Starship <https://starship.rs/>`_: focused on showing development contexts. Probably the largest set of detected languages.
+- `Spaceship <https://spaceship-prompt.sh/>`_: similar spirit to Starship (with a little bit fewer features), but only for Zsh.
+- `Oh-My-Posh <https://ohmyposh.dev/>`_: tries to support a little bit of everything, and thus does not really shine at something. Only for Zsh.
+- `Powerlevel10k <https://github.com/romkatv/powerlevel10k>`_: the most popular on Github, focused on fancy theming, but to the expense of the UX.
+- `Powerline <https://github.com/powerline/powerline>`_: primarily a daemon-based status line generator, hence with good support for latency-expensive features. Probably the largest feature set for various services status.
+- `Pure <https://github.com/sindresorhus/pure>`_: quite popular, despite focusing on being minimalist (only supports Zsh, Git, and a few shell features).
+
+The following table compares those prompts systems in details.
+
+In that table, the numbers in cells figure the level of quality of the feature.
+*Popularity* is the sum of levels in the row.
+*Support* lines are the sum of levels in the column, for each *category* section.
+*Category* sections are sorted from top to bottom based on their average popularity.
+Projects are sorted from left to right, based on their *support* score in the *essentials* section.
+
+.. image:: prompts_comparison.svg
+   :alt: A long table with 10 columns and 155 lines.
```

Comment:
Still needs some sort of key or explanation. What do the numbers mean? What is the "Popularity" column?

Also, all cells with text in them need a solid background, otherwise when viewing in a dark theme the text is unreadable.

Reply:
It is already explained on the paragraph before (I can put it as a note to make it clearer?).

Reply:
I must be blind, sorry.

> the numbers in cells figure the level of quality of the feature.

That seems quite arbitrary and subjective. At the very least we should document some guidelines for how we came up with those numbers, so we can try to be consistent in the future, and so anyone wanting more details on the chart can understand why we reached the numbers we did.

Reply:
And I still think there should be less transparency, this is what it looks like for me:
![image](https://user-images.githubusercontent.com/8375571/186209451-bf5878a9-e34c-40f7-9dbf-d416c6fcf979.png)